examples/src/common/camera.ts

```diff
@@ -1,8 +1,44 @@
-import type { box2D, vec2 } from '@alleninstitute/vis-geometry';
+import { Box2D, Vec2, type box2D, type vec2 } from '@alleninstitute/vis-geometry';
 
 // a basic camera, for viewing slices
 export type Camera = {
     readonly view: box2D; // a view in 'data space'
     readonly screen: vec2; // what that view projects to in display space, aka pixels
     readonly projection: 'webImage' | 'cartesian';
 };
+/**
+ * Zooms relative to your current mouse position
+ * @param view your current view
+ * @param screenSize the size of your canvas/screen
+ * @param zoomScale the scale you want to apply to your view
+ * @param mousePos the offsetX and offsetY of your mouse
+ */
+export function zoom(view: box2D, screenSize: vec2, zoomScale: number, mousePos: vec2) {
```

Comment:
other than the unhelpful docs, these are excellent - thanks for unifying the camera logic!